extensions/components/com_redcore/admin/views/webservice/tmpl/default.php

```diff
@@ -14,150 +14,12 @@
 JHtml::_('behavior.keepalive');
 JHtml::_('rbootstrap.tooltip');
 JHtml::_('rjquery.chosen', 'select');
+JHtml::script(JUri::root() . '/media/redcore/js/redcore.js');
```

Comment:
What folder should I put the js file in, so that it gets minified? I was wondering about that while refactoring it, but wasn't sure. I you tell me where it needs to go I'll take care of it. =^D